package.json

```diff
@@ -1,20 +1,22 @@
 {
-  "name": "@adobe/cordova-acpgriffon",
-  "version": "0.0.2",
+  "name": "@adobe/cordova-aepassurance",
+  "version": "0.0.1",
   "description": "Adobe Experience Platform - Project Griffon plugin for Cordova apps.",
   "cordova": {
-    "id": "cordova-acpgriffon",
+    "id": "cordova-aepassurance",
     "platforms": [
       "ios",
       "android"
     ]
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/adobe/cordova-acpgriffon"
+    "url": "https://github.com/adobe/cordova-aepassurance"
   },
   "keywords": [
     "adobe",
+    "aepassurance",
+    "assurance",
     "griffon",
```

Comment:
why griffon ?

Reply:
The product name is still Griffon,
So it could be a hint when people search with that name